### PR TITLE
Update documentation & some tests

### PR DESCRIPTION
**Pull request by vademo**
- Documentation
  - Licence
  - Test of example routes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,25 @@
-[![Build
-Status](https://travis-ci.com/digipolisantwerp/starter-kit_app_nodejs.svg?branch=master)](https://travis-ci.com/digipolisantwerp/starter-kit_app_nodejs)
-[![Coverage Status](https://coveralls.io/repos/github/digipolisantwerp/starter-kit_app_nodejs/badge.svg?branch=master)](https://coveralls.io/github/digipolisantwerp/starter-kit_app_nodejs?branch=master)
+[![Build Status](https://travis-ci.org/vademo/starter-kit_app_nodejs.svg?branch=master)](https://travis-ci.org/vademo/starter-kit_app_nodejs)
+[![Coverage Status](https://coveralls.io/repos/github/vademo/starter-kit_app_nodejs/badge.svg?branch=master)](https://coveralls.io/github/vademo/starter-kit_app_nodejs?branch=master)
 # starter-kit_app_nodejs
+
+A boilerplate project for Digipolis created to keep a consistent folderstucture and implements some much used modules like:
```
Comment (lievenvg) [inline README.md]: Nitpicking, but for readability I would add a comma. 

> ... Digipolis, created to ...

File: backend/src/controllers/examples.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
   getExampleById,
   getAllExamples,
 } from '../services/example';
 
 export function getExample(req, res, next) {
   getExampleById(req.params.exampleId)
-    .then(example => res.json(example))
+    .then((example) => {
+      if (!example) {
+        next(Errors.notFound({ message: `example with id ${req.params.exampleId} not found` }));
```
Comment (stefanpante) [inline backend/src/controllers/examples.js]: return next, anders wordt de res.json ook nog uitgevoerd, wat denk ik niet de bedoeling is?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,25 @@
...
+## How to use
+
+###Backend:
+
+ **[Backend documentation](backend/README.md)**
+
+### Run:
+|   |  cmd |  description |  
+|---|---|---|
+|**Run app in docker**| `docker-compose up`|Run app in dockerized enviroment|
```
Comment (lievenvg) [inline README.md]: For consistency reasons I would add some space here and check your spaces between `cmd` and `description`.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,25 @@
...
+
+
+### Testing:
+|   |  cmd |  description |  
+|---|---|---|
+|   **Run all tests in docker** | `docker-compose -f docker-compose.ci.yml up`| Run all tests
```
Comment (lievenvg) [inline README.md]: A bit too much space here as well. 😉

File: README.md [hunk 5]
```diff
@@ -1,4 +1,25 @@
-[![Build
-Status](https://travis-ci.com/digipolisantwerp/starter-kit_app_nodejs.svg?branch=master)](https://travis-ci.com/digipolisantwerp/starter-kit_app_nodejs)
-[![Coverage Status](https://coveralls.io/repos/github/digipolisantwerp/starter-kit_app_nodejs/badge.svg?branch=master)](https://coveralls.io/github/digipolisantwerp/starter-kit_app_nodejs?branch=master)
+[![Build Status](https://travis-ci.org/vademo/starter-kit_app_nodejs.svg?branch=master)](https://travis-ci.org/vademo/starter-kit_app_nodejs)
```
Comment (lievenvg) [inline README.md]: Doing some product placement here? 😉

File: backend/README.md [hunk 6]
```diff
@@ -1,16 +1,36 @@
-# Starter kit
+[![Build Status](https://travis-ci.org/vademo/starter-kit_app_nodejs.svg?branch=master)](https://travis-ci.org/vademo/starter-kit_app_nodejs)
```
Comment (lievenvg) [inline backend/README.md]: Product placement again?

File: backend/README.md [hunk 7]
```diff
@@ -1,16 +1,36 @@
...
+[![Coverage Status](https://coveralls.io/repos/github/vademo/starter-kit_app_nodejs/badge.svg?branch=master)](https://coveralls.io/github/vademo/starter-kit_app_nodejs?branch=master)
+# Starter kit node
 
 ## Usage
+### Instalation:
+- We use npm as our default package manager. to install run `npm install`
```
Comment (lievenvg) [inline backend/README.md]: Nitpicking again.

>  To install run `npm install`.

File: backend/README.md [hunk 8]
```diff
@@ -1,16 +1,36 @@
...
+
+### Run:
+|                       | cmd                       | description |
+| --------------------- | ------------------------- |---|
+| **Start app local**   | `npm start`               | Start app with local node|
```
Comment (lievenvg) [inline backend/README.md]: Just one more space at the end.

File: backend/README.md [hunk 9]
```diff
@@ -1,16 +1,36 @@
...
 By default the login routes are mounted on `/auth`:
 
  - **Login mprofiel:** 			`/auth/login/mprofiel`
- - **isloggedin mprofiel:**  	`/auth/isloggedin`
+ - **isloggedin mprofiel:**  	`/auth/login/mprofiel`
  - **logout mprofiel:** 		`/logout/callback/{service}`
 
 ## Testing
 
+|                               |  cmd                                          |  description |
```
Comment (lievenvg) [inline backend/README.md]: Check the spaces in the two titles.

File: backend/test/routes/example.test.js [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+      new Example({ firstName: 'Jonn' }).save((err, result) => {
+        // eslint-disable-next-line
+        id = result['_id'];
+        done();
+      });
+    });
+    after((done) => {
+      Example.findById(id).remove().exec(done);
+    });
+    it(`GET: ${routeExamples}/:id invalid id`, () => request(server)
```
Comment (lievenvg) [inline backend/test/routes/example.test.js]: Is this a invalid id?

File: backend/src/controllers/examples.js (hunk 2 above)
Comment (vademo) [reply backend/src/controllers/examples.js]: inderdaad, fixed

File: README.md (hunk 5 above)
Comment (vademo) [reply README.md]: Started from my fork my bad ;)

File: backend/test/routes/example.test.js (hunk 10 above)
Comment (vademo) [reply backend/test/routes/example.test.js]: nope, fixed

File: README.md (hunk 1 above)
Comment (vademo) [reply README.md]: fixed.

File: README.md (hunk 3 above)
Comment (vademo) [reply README.md]: fixed.

File: README.md (hunk 4 above)
Comment (vademo) [reply README.md]: fixed.

File: backend/README.md (hunk 6 above)
Comment (vademo) [reply backend/README.md]: fixed.

File: backend/README.md (hunk 7 above)
Comment (vademo) [reply backend/README.md]: fixed.

File: backend/README.md (hunk 9 above)
Comment (vademo) [reply backend/README.md]: fixed.

**Pull request merged**